content/en/docs/ops/ambient/install/helm-installation/index.md

```diff
@@ -65,6 +65,19 @@ The `ztunnel` chart installs the ztunnel DaemonSet, which is the node-proxy comp
 $ helm install ztunnel istio/ztunnel -n istio-system
 {{< /text >}}
 
+### (Optional) Install an ingress gateway
+
+{{< text syntax=bash snip_id=install_ingress >}}
+$ helm install istio-ingress istio/gateway -n istio-ingress --wait --create-namespace
+{{< /text >}}
```

Comment:
I think it makes more sense to have the snippet after the See... and warning.

Reply:
It does, and since I copied this block as-is from `content/en/docs/setup/install/helm/index.md` I'll fix it there as well.